package.json

```diff
@@ -57,6 +57,7 @@
     "emotion-theming": "^10.0.27",
     "i18next": "^19.3.3",
     "js-yaml": "^3.13.1",
+    "lodash": "^4.17.15",
```

Comment:
wow, I can't believe we didn't have this yet

Reply:
Yeah, it was only a transitive dep before.